kasa/tests/test_readme_examples.py

```diff
@@ -69,79 +70,109 @@ def test_discovery_examples(readmes_mock):
     """Test discovery examples."""
     res = xdoctest.doctest_module("kasa.discover", "all")
     assert res["n_passed"] > 0
-    assert res["n_warned"] == 0
+    if sys.version_info < (3, 13):
+        # xdoctest raises DeprecationWarning: 'count' is passed as positional argument
+        # in checker.py:  new_text = re.sub(blankline_pattern, '\n', text, re.MULTILINE)
+        assert res["n_warned"] == 0
```

Comment:
We should look into a solution for this, or add a separate >=3.13 check for `n_warned` for the time being.